modules/zed/hm.nix

```diff
@@ -0,0 +1,31 @@
+{
+  inputs,
+  config,
+  lib,
+  ...
+}: let 
+  theme = config.lib.stylix.colors {
+    templateRepo = config.lib.stylix.templates.tinted-zed;
+  };
+in {
+  options.stylix.targets.zed = {
+    enable = config.lib.stylix.mkEnableTarget "zed" true;
+  };
+
+  config = lib.mkIf (config.stylix.enable && config.stylix.targets.zed.enable && config.programs.zed-editor.enable) {
+    programs.zed-editor = {
+      userSettings = {
+        "theme" = lib.mkDefault "Base16 ${config.scheme.scheme-name}";
+
+        "ui_font_family" = lib.mkDefault config.stylix.fonts.sansSerif.name;
+        "ui_font_size" = lib.mkDefault config.stylix.fonts.sizes.applications;
+        "buffer_font_family" = lib.mkDefault config.stylix.fonts.monospace.name;
+        "buffer_font_size" = lib.mkDefault config.stylix.fonts.sizes.terminal;
+      };
```

Comment:
As already discussed in https://github.com/danth/stylix/pull/388, Stylix should not declare options with `lib.mkDefault`.